docs/HazardAnalysis/HazardAnalysis.tex

```diff
@@ -154,6 +155,44 @@ \section{Failure Mode and Effect Analysis}
       Error messages and logging & a. Allow administrator to adjust
       permissions & SR-AR1 \newline SR-AR4 & H3-2 \\
       \hline
+      Accept Contractor Document Upload & Document is valid but is uploaded in
+      the wrong location & Poor discoverability and organization &
+      a. Human-computer interface failure &User detects &
+      a. Alert administrator to adjust & FR1 & H4-1 \\
+      \cline{2-8}
+      & Document is malicious and should not be in the system & Malware
+      infection &a. Malicious user gained access to system & Error messages and
+      logging & a. Remove file from system and notify administrator & SR-AR1
+      & H4-2 \\
+      \cline{2-8}
+      & File chosen for upload is corrupted and not able to be opened &
+      User fails to open file &a. Failure in document chosen or upload process&
+      Error messages and logging & a. Remove file from system and
+      notify administrator & SR-IR3 & H4-3 \\
+      \cline{2-8}
+      & File type uploaded is not a supported type & System can't display
+      file correctly to the user & a. Failure in file type detection
+      \newline b. System permitted user to attempt upload of wrong file &
+      Error message and logging & a. Notify user of incompatible file type &
+      FR1 & H4-4 \\
+      \cline{2-8}
+      & The upload process is interrupted and the file upload is incomplete. &
+      File upload fails & a. System connection interruption &
+      Error messages and logging of file name & a. Notify user & PR-SC1 & H4-5\\
+      \hline
+      Contractor acknowledges document & Contractor does not acknowledge
+      correctly and it is not saved to the system & Acknowledgement not received
+      & a. User does not understand feedback from the application interface
+      \newline b. User does not agree to sign the document & Application detects
+      unsuccessful acknowledgement state & a. Notify user and log failure &
```

Comment:
disclude logging. Include who notifies user

Reply:
removed logging and specified the system notifies user